javascript-arrays/main.js

```diff
@@ -0,0 +1,30 @@
+var colors;
+colors = ['red',
```

Comment:
This isn't necessary in JavaScript, you can just assign a value to a variable on the same line you declare it